babelsberg/constraintinterpreter.js

```diff
@@ -509,7 +508,7 @@ Object.subclass('Constraint', {
     addConstraintVariable: function(v) {
         if (v && !this.constraintvariables.include(v)) {
             this.constraintvariables.push(v);
-            v.logTimings = this.logTimings;
+            v.logTimings = this._options.logTimings;
```

Comment:
Test show at this point `_options` does not exist yet. Neither did our custom flag but it didn't throw...